### ISSUE TITLE
Add instructions for portfolio, also correct API link?

### ISSUE THREAD

**Issue by RealWeeks** [ga-wdi-boston/game-project#52]
Need to clone api and push to their own heroku account.  Point front end at new api.

**Comment by gaand** [comment]
After they know what that means :smile:

**Comment by jrhorn424** [comment]
We weren't planning on taking down `ttt.wdi-bos.com`. It should be stable enough that they can use it for the meet and hire.

**Comment by gaand** [comment]
I don't want to maintain backwards compatibility.  @jrhorn424 We can for 010.  BTW, `ttt.wdibos.com` isn't what 010 used,  it's `tic-tac-toe.wdibos.com`.

**Comment by RealWeeks** [comment]
noted in commit 876a824bc2aa19dba661f088ff3f13830228ad92

Editted: leading `#` removed by @gaand

**Comment by jrhorn424** [comment]
How was this addressed? It isn't clear to me.

In any case, I'm reopening under a new title. Should hosting on your own be included as a guide? Included here as instructions?

Does the hosted API link (heroku) go on this prompt, or on the `game-project-api`. I lean toward the former, but barely.

**Comment by gaand** [comment]
Have the developers ask for the API URL?

**Comment by BenGitsCode** [comment]
Yes. First instance was here I believe https://github.com/ga-wdi-boston/game-project/issues/400

**Comment by gaand** [comment]
I prefer that they ask for the API link.  But, that's me.

**Comment by jrhorn424** [comment]
> I prefer that they ask for the API link. But, that's me.

How strong is your preference? Can we discuss in person? I'd like to see it from your POV.

**Comment by gaand** [comment]
@jrhorn424 I think it's a good lesson on asking when you don't know something.  

Want to chat about it on Tuesday?

**Comment by payne-chris-r** [comment]
I don't see a decision on this. I think it makes sense to post the `game-project-api` hosted link _somewhere_. Either in this repo or the api one (https://github.com/ga-wdi-boston/game-project-api).

**Comment by jrhorn424** [comment]
Let's put our heads together on this at the next leads meeting.

**Comment by MicFin** [comment]
Just adding a 👍  about discussing this.  @BenGitsCode @danman01 and I discussed this during Project Eval for `017`, had the same variety of views, and did not come to a consensus but talked about some pros and cons.

**Comment by jordanallain** [comment]
Would love to reach a conclusion on this!
